src/metrics/metrics.jl

```diff
@@ -63,6 +63,15 @@ function metric_name(f::Function)::String
     return join(split(String(Symbol(f))[2:end], "_"), " ")
 end
 
+"""
+    name_symbol(m::Metric)::String
+
+Get name of metric as a symbol.
+"""
+function name_symbol(m::Metric)::Symbol
```

Comment:
Suggest:

```suggestion
function metric_name_as_symbol(m::Metric)::Symbol
```

or maybe rely on multiple dispatch and make it simply:

```suggestion
function as_symbol(m::Metric)::Symbol
```

Reply:
I like `as_symbol`, it makes sense to me. But before deciding, I have a question: there is another function above this one called `metric_name`. So, first of all: relying on multiple dispatch we could also call it `name`, as it receives a metric as its only argument. But looking at these two functions side by side, they return "the same thing" but one as a string and other as a symbol. So we could

1. Have the string one be called `name` and this `as_symbol`. 
2. Have them be called `as_string` and `as_symbol`.
3. Have one be called `name_string` and other `name_symbol`

What do you think?

Reply:
I vote for `as_string` and `as_symbol`.